src/Network/Network.Test/ScenarioTests/PrivateLinkServiceTests.ps1

```diff
@@ -118,3 +118,87 @@ function Test-PrivateLinkServiceCRUD
     }
 } 
 
+<#
+.SYNOPSIS
+Test operation for PrivateEndpointConnection.
+#>
+function Test-PrivateEndpointConnectionCRUD
+{
+    # Setup
+    $rgname = Get-ResourceGroupName;
+    $rname = Get-ResourceName;
+    $location = Get-ProviderLocation "Microsoft.Network/privateLinkServices" "westus2";
+    # Dependency parameters
+    $IpConfigurationName = "IpConfigurationName";
+    $vnetName = Get-ResourceName;
+    $ilbFrontName = "LB-Frontend";
+    $ilbBackendName = "LB-Backend";
+    $ilbName = Get-ResourceName;
+
+    $serverName = "server-$(Get-Random)"
+    $serverLogin = "testusername"
+    <#[SuppressMessage("Microsoft.Security", "CS002:SecretInNextLine", Justification="Test passwords only valid for the duration of the test")]#>
+    $serverPassword = "t357ingP@s5w0rd!Sec"
+    $credentials = new-object System.Management.Automation.PSCredential($serverLogin, ($serverPassword | ConvertTo-SecureString -asPlainText -Force))
+    $databaseName = "mySampleDatabase"
+    $peName = "mype"
+
+    try
+    {
+        $resourceGroup = New-AzResourceGroup -Name $rgname -Location $location;
+
+        if ((Get-NetworkTestMode) -ne 'Playback')
+        {
+            # Create Sql Storage Account
+            $server = New-AzSqlServer -ResourceGroupName $rgname `
+                -ServerName $serverName `
+                -Location $location `
+                -SqlAdministratorCredentials $credentials
+
+            $database = New-AzSqlDatabase  -ResourceGroupName $rgname `
+                -ServerName $serverName `
+                -DatabaseName $databaseName `
+                -RequestedServiceObjectiveName "Basic" `
+                -Edition "Basic"
+
+            $sqlResourceId = $server.ResourceId
+        }
+        else
+        {
+            $sqlResourceId = "/subscriptions/e05dbbce-79c2-45a2-a7ef-f1058856feb3/resourceGroups/ps3301/providers/Microsoft.Sql/servers/server-841315352"
```

Comment:
I don't think it's a good idea to put usename, password, subscription and other sensitive info in the code.

Reply:
@wyunchi-ms 
This is for testing purpose and it will create resource group and all other resource and delete after test.
Also, this same code  snippet is the same as in 
\azure-powershell\src\Sql\Sql.Test\ScenarioTests\Common.ps1
```
<#
.SYNOPSIS
Creates the basic test environment needed to perform the Sql data security tests - resource group, server and database
#>
function Create-BasicTestEnvironmentWithParams ($params, $location, $serverVersion)
{
	New-AzResourceGroup -Name $params.rgname -Location $location
	$serverName = $params.serverName
	$serverLogin = "testusername"
	<#[SuppressMessage("Microsoft.Security", "CS002:SecretInNextLine", Justification="Test passwords only valid for the duration of the test")]#>
	$serverPassword = "t357ingP@s5w0rd!Sec"
	$credentials = new-object System.Management.Automation.PSCredential($serverLogin, ($serverPassword | ConvertTo-SecureString -asPlainText -Force))
	New-AzSqlServer -ResourceGroupName $params.rgname -ServerName $params.serverName -Location $location -ServerVersion $serverVersion -SqlAdministratorCredentials $credentials
	New-AzSqlDatabase -DatabaseName $params.databaseName -ResourceGroupName $params.rgname -ServerName $params.serverName -Edition Basic
}
```
So I don't think there is any harm or concern on this passcode.
Thanks.

Reply:
This would work only once. Resource group name and server name are randomized, and the different subscription could be used. If you need sqlResourceId - construct it from variables

Reply:
Got it.